01.fizzbuzz/fizzbuzz.rb

```diff
@@ -1,16 +1,12 @@
-a=1
-
-20.times{
-
-if a%15 == 0 
-  puts "fizzbuzz"
-elsif a%5 ==0
-  puts "buzz"
-elsif a%3 ==0
-  puts "fizz"
-else 
-  puts a
+20.times do |n|
+    if (n+1) % 15 == 0
```

Comment:
```suggestion
  if (n + 1) % 15 == 0
```

Rubyのインデントはスペース2文字が一般的です。
`+`も演算子なので前後にスペースを入れましょう。